CONTRIBUTING.md

```diff
@@ -23,8 +23,8 @@ For example, `added-ultra-fast-flashing.md` would lead to an entry in the `## Ad
 
 ## How to build cargo-embed/ cargo-flash from source
 
-`cargo-embed` is a so called [cargo subcommand](https://doc.rust-lang.org/book/ch14-05-extending-cargo.html). It is a programm named `cargo-embed` which is installed in the users path. Thus when applying some small fixes to `cargo-embed` you can run `cargo build --features cli` and then use the executable in the target folder named cargo-embed directly. You can also use [cargo install --path probe-rs --features cli](https://doc.rust-lang.org/cargo/commands/cargo-install.html) to install your current checkout locally overriding what you previously had installed using `cargo install cargo-embed`.
+`cargo-embed` is a so called [cargo subcommand](https://doc.rust-lang.org/book/ch14-05-extending-cargo.html). It is a program named `cargo-embed` which is installed in the users path. Thus when applying some small fixes to `cargo-embed` you can run `cargo build --features cli` and then use the executable in the target folder named cargo-embed directly. You can also use [cargo install --path probe-rs-tools](https://doc.rust-lang.org/cargo/commands/cargo-install.html) to install your current checkout locally overriding what you previously had installed using `cargo install cargo-embed`.
```

Comment:
```suggestion
`cargo-embed` is a so called [cargo subcommand](https://doc.rust-lang.org/book/ch14-05-extending-cargo.html). It is a program named `cargo-embed` which is installed in the users path. Thus when applying some small fixes to `cargo-embed` you can run `cargo build` and then use the executable in the target folder named cargo-embed directly. You can also use [cargo install --path probe-rs-tools](https://doc.rust-lang.org/cargo/commands/cargo-install.html) to install your current checkout locally overriding what you previously had installed using `cargo install cargo-embed`.
```

Reply:
Good point. Also a few refs in the website: 
https://github.com/probe-rs/webpage/pull/118

Additionally, do any of these in `Cargo.toml` need to change?
https://github.com/probe-rs/probe-rs/blob/master/probe-rs/Cargo.toml#L115-L116

Reply:
> Additionally, do any of these in Cargo.toml need to change?

I don't know. @Yatekii ?

Reply:
Yup, that needs to change afaik.

Reply:
I thought so, but from this PR it seems that `probe-rs-cli` is the intended binary crate:
https://github.com/probe-rs/probe-rs/pull/2364

Reply:
The PR comment was not updated properly. The binary crate is named probe-rs-tools and we don't actually plan on releasing it on cargo. It will be available with ready built binaries through GH :)